### PR TITLE
config: enable tars for tidb dashboard

### PR DESCRIPTION
**Pull request by hi-rustin**
I found that tidb-dashboard was not using tichi to merge PRs, and after checking, I found that tichi could not automatically merge PRs because the repository did not enable the tars plugin to help merge bases. so this PR will enable tars for tidb-dashboard.

### CODE REVIEW

**Comment by hi-rustin** [comment]
cc: @breeswish

**Comment by hi-rustin** [comment]
/hold

Awaiting approval from SIG-diagnosis(@breeswish).

I don't remember why I didn't configure tars for you guys, sorry for that.

**Comment by hi-rustin** [comment]
Oops! I missed some configuration for the plugin. I will fix it later.

**Review by breeswish — APPROVED** [review]

**Comment by hi-rustin** [comment]
/unhold

/assign @Mini256 

PTAL~ Thanks!

**Review by Mini256 — APPROVED** [review]

**Pull request merged**
